schedule/yast/sles+sdk+proxy_SCC_via_YaST_Online_pvm.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: sles+sdk+proxy_SCC_via_YaST
+description: >
+  Add add-on via YaST control center directly https://progress.opensuse.org/issues/16402
+vars:
+  DESKTOP: gnome
+  SCC_ADDONS: sdk
+  SCC_REGISTER: yast
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_scc
+  - installation/module_registration/register_module_desktop
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+  - installation/partitioning/accept_proposed_layout
+  - installation/clock_and_timezone/accept_timezone_configuration
+  - installation/authentication/use_same_password_for_root
+  - installation/authentication/enable_autologin
```

Comment:
we don't need to schedule this module for autologin. example:
https://openqa.suse.de/tests/8752098#step/default_user_simple_pwd/4
it is not default in SLE, only in openSUSE.

Reply:
OK, I see, I'll remove the autologin module, thanks.